### PR TITLE
Windows shortcut cheat sheet created

### PR DESCRIPTION
**Pull request by Sloff**
**What does your Instant Answer do?**
It shows a cheat sheets of all the windows shortcuts

**What is the data source for your Instant Answer? (Provide a link if possible)**
[Wikipedia](https://en.wikipedia.org/wiki/Table_of_keyboard_shortcuts)

**Why did you choose this data source?**
It's very extensive and regularly updated

**Are there any other alternative (better) data sources?**
[Windows Help](http://windows.microsoft.com/en-us/windows/keyboard-shortcuts#keyboard-shortcuts=windows-8), but it is specific to a certain version of windows

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Shortcut lovers

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
capture

-----
IA Page: https://duck.co/ia/view/microsoft_windows_cheat_sheet

### CODE REVIEW

**Comment by javathunderman** [comment]
Cool, I was working on this but I got caught up with other stuff. @moollaza This LGTM, though I suggest someone else give a look before the merge. :+1:

File: share/goodie/cheat_sheets/windows-shortcuts.json [hunk 1]
```diff
@@ -0,0 +1,582 @@
+{
+    "id": "windows_shortcuts_cheat_sheet",
+    "name": "Windows Shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/windows-shortcuts.json]: Let's change this to "Microsoft Windows"

File: share/goodie/cheat_sheets/windows-shortcuts.json [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+    "description": "",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/windows-shortcuts.json]: I think we should explain which OS versions this applies to

File: share/goodie/cheat_sheets/windows-shortcuts.json [hunk 3]
```diff
@@ -0,0 +1,582 @@
+{
+    "id": "windows_shortcuts_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/windows-shortcuts.json]: Let's change this to `microsoft_windows_cheat_sheet`

Comment (moollaza) [inline share/goodie/cheat_sheets/windows-shortcuts.json]: Please change the filename as well: `microsoft-windows.json`

File: share/goodie/cheat_sheets/windows-shortcuts.json [hunk 4]
```diff
@@ -0,0 +1,582 @@
...
+        }, {
+            "val": "Make selected text bulleted or Numbered Items",
+            "key": "[Ctrl] [Shift] [l]"
+        }, {
+            "val": "Insert Linebreak",
+            "key": "[Shift] [Enter]"
+        }],
+        "Browsers": [{
+            "val": "Go to Address Bar",
+            "key": "[Alt] [d]/[c]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/windows-shortcuts.json]: Please make sure sub-alternatives like this are wrapped in parentheses: `[Alt] ([d]/[c])`

**Comment by Sloff** [comment]
Hi @moollaza, changes made as suggested

**Comment by moollaza** [comment]
@Sloff thanks! LGTM :+1:

**Comment by moollaza** [comment]
@Sloff congrats! It's live: https://duckduckgo.com/?q=microsoft+windows+cheat+sheet&t=ffab&ia=cheatsheet

Sorry for the delay!

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by Sloff** [comment]
Awesome, thanks @moollaza

**Pull request merged**